packages/repack/commands.js

```diff
@@ -50,6 +50,117 @@ function getCommands() {
   return commands;
 }
 
+const startCommandOptions = [
+  {
+    name: '--cert <path>',
+    description: 'Path to custom SSL cert',
+  },
+  {
+    name: '--host <string>',
+    description: '(default: "")',
+  },
+  {
+    name: '--http',
+    description: 'Enables https connections to the server',
+  },
+  {
+    name: '--key <path>',
+    description: 'Path to custom SSL key',
+  },
+  {
+    name: '--port <number>',
+    description: 'The port number that runs the server on',
+  },
+  {
+    name: '--no-interactive',
+    description: 'Disables interactive mode',
+  },
+  {
+    name: '--silent',
+    description: 'Silents all logs to the console/stdout',
+  },
+  {
+    name: '--experimental-debugger',
+    description:
+      "[Experimental] Enable the new debugger experience and 'j' to debug. This enables the new frontend experience only: connection reliability and some basic features are unstable in this release.",
```

Comment:
not sure we support `j` to debug here, can we double check on this?

Reply:
looks like we just support experimental debugger without handling 'j' key

Reply:
so maybe let's add support for handling `j` in separate PR, I think it won't be a big effort but will sync behaviour with Metro dev server 👍

Reply:
ok, let's remove the 'j' to debug for now, and we will add that in a separate PR